code/modules/mob/living/simple_animal/friendly/corgi.dm

```diff
@@ -651,15 +651,16 @@
 
 /mob/living/simple_animal/pet/corgi/Ian/borgi/Life(seconds, times_fired)
 	..()
-	if(emagged && prob(25))
-		var/mob/living/carbon/target = locate() in view(10,src)
-		if(target)
-			shootAt(target)
-
 	//spark for no reason
 	if(prob(5))
 		do_sparks(3, 1, src)
 
+/mob/living/simple_animal/pet/corgi/Ian/borgi/handle_automated_action()

```

Comment:
Again, unsure if pet mobs use AI_ON and thus use handle_automated_action()

Reply:
See above.